internal/controllers/vgmanager/lvmd/lvmd.go

```diff
@@ -31,6 +32,50 @@ const (
 	maxReadLength            = 2 * 1 << 20 // 2MB
 )
 
+func DeepCopyConfig(c *Config) *Config {
```

Comment:
Why do you need this? Couldnt you just deref and reref the Config? Or use deepcopy gen?